find-dialog.hpp

```diff
@@ -83,14 +83,23 @@ class FindDialog : public QDialog
     */
     FindDialog(AeraVisualizerWindow* parentWindow, ReplicodeObjects& replicodeObjects);
 
+    // The stepEvent and unstepEvent functions in AeraVisualizerWindow use this to notify the
+    // find dialog that it should refresh its matches (don't put much here since it's called A LOT)
+    void reportStepEvent() {
+      timeSteppedFlag_ = true;
+      highlightAllFlag_ = true;   // This'll need to be redone as well
```

Comment:
I'm curious what you mean by "This'll need to be redone as well". Do we need more explanation in the comment, for the record?

Reply:
That just means that the highlight all effect will need to be reapplied when events are stepped. I'll go ahead and remove it, though, it is a bit ambiguous